app/_locales/en/messages.json

```
@@ -5216,6 +5216,18 @@
   "toggleEthSignOn": {
     "message": "ON (Not recommended)"
   },
+  "toggleRequestQueueDescription": {
+    "message": "This allows you to select a network for each site instead of a single selected network for all sites. This feature will prevent you from switching networks manually, which my break your user experience on certain sites."
+  },
+  "toggleRequestQueueField": {
+    "message": "Select networks for each site"
+  },
+  "toggleRequestQueueOff": {
+    "message": "off"
+  },
+  "toggleRequestQueueOn": {
+    "message": "on"
```

```suggestion
  "toggleRequestQueueOff": {
    "message": "Off"
  },
  "toggleRequestQueueOn": {
    "message": "On"
```


just want to make sure casing for this correct. I don't seem to see other lowercased 'off' or 'on'

---

yeah some of the others have all caps, some cap case. Ill take it tho.